src/alliteratedActors.js

```diff
@@ -22,7 +22,8 @@
  * @return {string}
  */
 function alliteratedActors(names) {
-  // write code here
+  return names
+    .filter(item => item.split(' ').every(name => name[0] === item[0]))[0];
```

Comment:
метод `find` вернет первое совпадение, фильтер не нужен
```suggestion
    .find(item => item.split(' ')
    .every(name => name[0] === item[0]));
```